CHANGELOG.md

```diff
@@ -5,10 +5,53 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [0.6.0] - 2021-07-08
+### Added
+* 10 new transforms, a masked loss wrapper, and a network adaptor for transfer learning
+* APIs to load networks and pre-trained weights from Clara Train Medical Model ARchives (MMARs)
+* Base metric and cumulative metric APIs, 4 new regression metrics
+* Initial CSV dataset support
+* Decollating mini-batch as the default first postprocessing step
+* Initial backward compatibility support via `monai.utils.deprecated`
+* Attention-based vision modules and `UNETR` for segmentation
+* Generic module loaders and Gaussian mixture models using the PyTorch JIT compilation
+* Inverse of image patch sampling transforms
+* Network block utilities `get_[norm, act, dropout, pool]_layer`
+* `unpack_items` mode for `apply_transform` and `Compose`
+* New event `INNER_ITERATION_STARTED` in the deepgrow interactive workflow
+* `set_data` API for cache-based datasets to dynamically update the dataset content
+* Fully compatible with PyTorch 1.9
+* `--disttests` and `--min` options for `runtests.sh`
+* Initial support of pre-merge tests with Nvidia Blossom system
+### Changed
+* Base Docker image upgraded to `nvcr.io/nvidia/pytorch:21.06-py3` from
+  `nvcr.io/nvidia/pytorch:21.04-py3`
+* Optionally depend on PyTorch-Ignite v0.4.5 instead of v0.4.4
```

Comment:
But all the handlers, engines, etc. still optional import ignite v0.4.4. Maybe we can say "compatible with ignite >= 0.4.4"?

Reply:
the support is actually dropped because we don't have tests for 0.4.4 anymore